config/settings.yml

```diff
@@ -76,6 +76,8 @@
   :concurrent_per_host: 1
   :scan_via_host: true
   :use_vim_broker: true
+:container_scanning:
+  :concurrent_per_ems: 1
```

Comment:
Are we running the actual scanning in the smart proxy worker (aka coresident_miqproxy?)

If so, I'd add concurrent_container_scans_per_ems to the coresident_miqproxy section.